http.bridge/itest/pom.xml

```diff
@@ -83,23 +82,23 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
-                <version>2.9</version>
-                <configuration>
-                    <artifactItems>
-                        <artifactItem>
-                            <groupId>org.kaazing</groupId>
-                            <artifactId>netx.http.bridge</artifactId>
-                            <version>${project.version}</version>
-                            <type>jar</type>
-                            <destFileName>netx.http.bridge.jar</destFileName>
-                        </artifactItem>
-                    </artifactItems>
-                </configuration>
                 <executions>
                     <execution>
+                        <phase>package</phase>
```

Comment:
Looks like [default phase](http://maven.apache.org/plugins/maven-dependency-plugin/copy-mojo.html) is process-sources, which I assume is what it was on before.  I also don't think it should be necessary to do this as maven is convention over configuration so default configuration should (most likely) be correct, unless we are doing something peculiar (read, potentially non standard and wrong)